projects/mci/inc/motor_controller.h

```diff
@@ -1,8 +1,13 @@
 #pragma once
-#include "gpio.h"
 
+#include "exported_enums.h"
+#include "generic_can.h"
+#include "pedal_rx.h"
 #include "precharge_control.h"
 
 typedef struct MotorControllerStorage {
   PrechargeStorage precharge_storage;
+  PedalRxStorage pedal_storage;
+  EEDriveOutput drive_state;
```

Comment:
I think having the drive_fsm module expose this via a global variable would be better, than to keeping it here. Similar to how you get pedal values now...